docs/release-notes.rst

```diff
@@ -6,6 +6,177 @@
+   any remaining references from configuration files and use ``resource_pool`` instead.
+
+**New Features**
+
+-  WebUI/CLI/API: Allow admins to manage resource quotas for auto-created namespaces directly.
+
+   -  WebUI: Add a "Namespace Bindings" section to the Create and Edit Workspace modals.
+
+      -  Users can input a namespace for a Kubernetes cluster. If no namespace is specified, the
+         workspace will be bound to the ``resource_manager.default_namespace`` field in the master
+         configuration YAML or the default Kubernetes namespace.
+
+      -  In the enterprise edition, users can auto-create namespaces and set resource quotas,
+         limiting GPU requests for that workspace. The Edit Workspace modal displays the lowest GPU
+         limit resource quota within the bound namespace.
+
+      -  Once saved, all workloads in the workspace will be sent to the bound namespace. Changing
+         the binding will affect future workloads, while in-progress workloads remain in their
+         original namespace.
+
+      -  For help with workspace-namespace bindings, visit :ref:`Manage Workspace-Namespace Bindings
+         <k8s-resource-caps>`.
+
+   -  CLI: Add new commands for creating and managing workspace namespace bindings.
+
+      -  Allow creating namespace bindings during workspace creation with ``det w create
+         <workspace-id> --namespace <namespace-name>`` or later with ``det w bindings set
+         <workspace-id> --namespace <namespace-name>``.
+
+      -  In the enterprise edition, users can use additional arguments ``--auto-create-namespace``
+         and ``--auto-create-namespace-all-clusters`` for auto-created namespaces. Set resource
+         quotas during workspace creation with ``det w create <workspace-name> --cluster-name
+         <cluster-name> --auto-create-namespace --resource-quota <resource-quota>``, or later with
+         ``det w resource-quota set <workspace-id> <quota> --cluster-name <cluster-name>``.
+
+      -  Add a command to delete namespace bindings with ``det w bindings delete <workspace-id>
+         --cluster-name <cluster-name>``.
+
+      -  Add a command to list bindings for a workspace with ``det w bindings list
+         <workspace-name>``. The ``--cluster-name`` field is required only for MultiRM setups.
+
+   -  API: Add new endpoints for creating and managing workspace namespace bindings.
+
+      -  Add POST and DELETE endpoints to ``/api/v1/workspaces/{workspace_id}/namespace-bindings``
+         for setting and deleting workspace namespace bindings.
+      -  Add a GET endpoint ``/api/v1/workspaces/{id}/list-namespace-bindings`` to list namespace
+         bindings for a workspace.
+      -  Add a POST endpoint ``/api/v1/workspaces/{id}/set-resource-quota`` to set resource quotas
+         on workspaces bound to auto-created namespaces.
+      -  Add a GET endpoint ``/api/v1/workspaces/{id}/get-k8-resource-quotas`` to retrieve enforced
```

Comment:
Add a GET endpoint ``/api/v1/workspaces/{id}/get-k8s-resource-quotas`` to retrieve enforced Kubernetes resource quotas for bound namespaces